spec/mana/engines/llm_spec.rb

```
@@ -0,0 +1,79 @@
+      expect(described_class.handler_stack).to be_empty
+    end
+  end
+
+  describe "#handle_mock" do
+    it "matches and returns stubbed values" do
+      Mana.mock!
+      Mana.current_mock.prompt("test", value: 42)
+
+      b = binding
+      engine = described_class.new(b)
+      result = engine.handle_mock("test prompt")
```

RuboCop (1.84.2)

[warning] Useless assignment to variable - `result`.

(Lint/UselessAssignment)